packages/manager/modules/pci/src/components/project/empty/empty.html

```diff
@@ -0,0 +1,43 @@
+<div class="container-fluid px-0 pci-project-empty">
+    <div class="row">
+        <div class="col-md-3">
+            <div class="p-5 placeholder">
+                <img data-ng-if="$ctrl.imageSource"
+                    data-ng-src="{{:: $ctrl.imageSource }}"
+                    data-ng-attr-alt="{{:: $ctrl.imageAlt }}"
+                    alt=""
```

Comment:
```suggestion
```
?

Reply:
Due to [HTMLHint configuration](https://github.com/ovh-ux/manager/blob/master/.htmlhintrc#L2) and `ngAttr` usage, it will report an error.

Reply:
😅 

okay, it didn't make sense as there was a `ngAttrAlt` but I get it now